lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -33,7 +33,7 @@ def persistence_safe_fallback():
     except DatabaseError:
         # Error deliberately logged, then swallowed. It's assumed that the wrapped code is not guaranteed to succeed.
         # TODO: enqueue a celery task to finish the task asynchronously, see TNL-5471
-        log.warning("Persistent Grades: Persistence Error, falling back.\n{}".format(format_exc()))
+        log.error("Persistent Grades: database_error.safe_fallback.\n{}".format(format_exc()))
```

Comment:
What was the original reason we had decided to "warn" here? This seems like something we'd want to include in the alert pattern, right?